src/ignitus-Internals/index.js

```diff
@@ -24,8 +24,16 @@ export class ErrorBoundary extends React.Component {
     if (errorInfo) {
       return (
         <div className="container">
-          <h2>Something went wrong.</h2>
-          <details style={{ whiteSpace: 'pre-wrap' }}>
+          <h2
+           style={{ textAlign: 'center', marginTop: '3em', color: '#000066' }}>
+            Unable to load the contributors!
+          </h2>
+          <h5 style={{ textAlign: 'center', marginTop: '20px' }}>
```

Comment:
Please avoid inline-styling try to write css as a part of styles file.

Reply:
> Please avoid inline-styling try to write css as a part of styles file.

okay I will take care of it